README.md

```diff
@@ -224,14 +222,14 @@ If a Lad strikes the final center nail and finishes it, that player is removed f
 If a Lord strikes a Lad's nail, the Lad may choose one of the following actions:
 
 * Do nothing
-* Get a free strike at the Uprising nail before play continues (the extra flip for striking the nail does not need to be performed)
+* Get a free strike at an Uprising nail before play continues (the extra flip for striking the nail does not need to be performed)
 * The Lad may reset their nail
 
 ### IX.D - A Lord illegally strikes an Uprising nail
 
-As mentioned in the Uprising rules, a Lord may call "Uprising" at the start of their turn and perform an Uprising flip similar to how a Lad would (including the extra flip). If this does not result in the nail being finished, or the necessary number of flips are not performed, that Lord is instantly demoted back to Lad and must start a new nail.
+As mentioned in the Uprising rules, a Lord may call "Uprising" at the start of their turn and perform an Uprising flip similar to how a Lad would (including the extra flip). If this does not result in the final Uprising nail being finished, or the necessary number of flips are not performed, that Lord is instantly demoted back to Lad and must start a new nail.
 
 ### IX.E - A Lad illegally strikes an Uprising nail
 
-If a Lad does not call "Uprising" or does not perform the necessary number of flips, the Uprising nail is reset along with that Lad's nail (they must hammer their current starting nail into the stump if it is not already finished before beginning a new one).
+If a Lad does not call "Uprising" or does not perform the necessary number of flips, all Uprising nails are reset along with that Lad's nail (they must hammer their current starting nail into the stump if it is not already finished before beginning a new one).
```

Comment:
>they must hammer their current starting nail
maybe also add remove as on option? Or also just leave as is if it's already a brand new nail that has never been hit?

Reply:
Updated it to point to the Resetting Nails section which explains better